README.md

```diff
@@ -93,7 +93,7 @@ const styles = StyleSheet.create({
 | data                   |        _not required if children is used_         | `array`  | Data to use in renderItem                             |
 | children               |                         -                         |  `node`  | Children elements                                     |
 | renderItem             |        _not required if children is used_         |  `func`  | Takes an item from data and renders it into the list. |
-| onMomentumScrollEnd    |                         -                         |  `func`  | Called after scroll end                               |
+| onMomentumScrollEnd    |                         -                         |  `func`  | Called after scroll end, passes new index             |
```

Comment:
Please update to: Called after scroll end and the first parameter is the current index